README.md

```diff
@@ -30,7 +30,9 @@ The
 [`api/prometheus` directory](https://github.com/prometheus/client_golang/tree/master/api/prometheus)
 contains the client for the
 [Prometheus HTTP API](http://prometheus.io/docs/querying/api/). It allows you
-to write Go applications that query time series data from a Prometheus server.
+to write Go applications that query time series data from a Prometheus
+server. It is in an early stage and further development is currently on hold
```

Comment:
That we're not developing isn't the same as not reviewing PRs. I presume at least one of us has review capacity if someone started fleshing it out.

> And how does the "it's alpha" not reflect on the whole client with the same argument you used above? 

It does, but being alpha is much better than a part of the project being unmaintained and thus dead.